args.go

```diff
@@ -0,0 +1,111 @@
+package main
+
+import (
+	"net/url"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	rootName    = "upgrade_manager"
+	genesisDir  = "genesis"
+	upgradesDir = "upgrades"
+	currentLink = "current"
+)
+
+// Config is the information passed in to control the daemon
+type Config struct {
+	Home     string
+	Name     string
+	Download bool
```

Comment:
Maybe rename this `AllowDownloadBinaries` or something that is a little clearer?